docs/content/preview/yugabyte-platform/alerts-monitoring/slow-queries-dashboard.md

```diff
@@ -39,26 +39,26 @@ The following table describes the **Slow Queries** column values.
 
```

Comment:
Comment for line 26-39:
In the above table, can we update the column names to match the field names in the new query info side panel:
![image](https://github.com/yugabyte/yugabyte-db/assets/88681329/ffbab16b-51da-4a2e-9d9a-9a3004c94a67)